openmetadata-ui/src/main/resources/ui/src/pages/ServiceDetailsPage/ServiceMainTabContent.tsx

```diff
@@ -260,16 +273,29 @@ function ServiceMainTabContent({
                             DEFAULT_SERVICE_TAB_VISIBLE_COLUMNS
                           }
                           extraTableFilters={
-                            <span>
-                              <Switch
-                                checked={showDeleted}
-                                data-testid="show-deleted"
-                                onClick={onShowDeletedChange}
-                              />
-                              <Typography.Text className="m-l-xs">
-                                {t('label.deleted')}
-                              </Typography.Text>
-                            </span>
+                            <>
+                              <span>
+                                <Switch
+                                  checked={showDeleted}
+                                  data-testid="show-deleted"
+                                  onClick={onShowDeletedChange}
+                                />
+                                <Typography.Text className="m-l-xs">
+                                  {t('label.deleted')}
+                                </Typography.Text>
+                              </span>
+
+                              {entityType === EntityType.DATABASE_SERVICE && (
+                                <Button
+                                  className="text-primary p-0"
+                                  data-testid="bulk-edit-table"
+                                  icon={<Icon component={IconEdit} />}
+                                  type="text"
+                                  onClick={handleEditTable}>
+                                  {t('label.edit')}
+                                </Button>
+                              )}
```

Comment:
i can see this getting repeated in few places, move it to a separate component or util